### PR TITLE
.NET Core Guide - minor list punctuation fixes, small wording change

### PR DESCRIPTION
**Pull request by bradumbaugh**
## Summary

- Added semi-colons (re: https://docs.microsoft.com/style-guide/punctuation/semicolons)
- Added two missing commas
- Changed "can be used in" to "can be used to build"

### CODE REVIEW

File: docs/core/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ ms.custom: "updateeachrelease"
 ---
 # .NET Core Guide
 
-[.NET Core](about.md) is an [open-source](https://github.com/dotnet/coreclr/blob/master/LICENSE.TXT) general-purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It's cross-platform, supporting Windows, macOS and Linux, and can be used in device, cloud, and IoT applications.
+[.NET Core](about.md) is an [open-source](https://github.com/dotnet/coreclr/blob/master/LICENSE.TXT), general-purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It's cross-platform; supports Windows, macOS, and Linux; and can be used to build device, cloud, and IoT applications.
```
Comment (rpetrusha) [inline docs/core/index.md]: - What makes .NET Core cross-platform is that it supports Windows, macOS, and Linux,; so cross-;platform and the list of platforms shouldn't be separated. The original sentence, with ", supporting" is correct. Please revert the change.
- the semicolon after Linux should be a comma. Please revert that change as well.

**Review by rpetrusha — APPROVED** [review]
Thanks for contributing to the dotnet/docs repo, @bradumbaugh, and for clarifying this sentence. I liked the addition of the comma after "open-source" and the replacement of "in" with "to build", but thought the other changes should be reverted.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Sorry, @bradumbaugh; I seem to have inadvertently approved your PR when I'd intended to request changes.

**Comment by rpetrusha** [comment]
Thanks for the rapid response, @bradumbaugh. I think that your last proposed version is the clearest of all.

**Comment by bradumbaugh** [comment]
@rpetrusha – made that change. Thanks for the help.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @bradumbaugh.  I'll merge your PR now.
